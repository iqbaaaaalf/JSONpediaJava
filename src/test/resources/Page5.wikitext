{{Redirect|National Hymn|the hymn tune of that title|God of Our Fathers}}
{{Two other uses|an American patriotic song|the US coin series|America the Beautiful Quarters|the documentary film|America the Beautiful (film)}}
{{Infobox song <!-- See Wikipedia:WikiProject_Songs -->
| Name           = America the Beautiful
| Cover          = Americathebeautiful.jpg
| Caption        = Commemoration plaque atop [[Pikes Peak]]
| Type           = [[Patriotic song|Patriotic]]
| Artist         =
| alt Artist     =
| Album          =
| Published      = 1895 (poem)
| Released       =
| track_no       =
| Recorded       =
| Genre          =
| Length         =
| Writer         = [[Katharine Lee Bates]]
| Composer       = [[Samuel A. Ward]]
| Label          =
| Producer       =
| Audio sample?  = yes
| Chart position =
| Tracks         =
| prev           =
| prev_no        =
| next           =
| next_no        =
| Misc           =
{{Audiosample
|Upper caption = America the Beautiful MIDI recording.
|Audio file    = America the Beautiful.mid
|Background    = Song
}}
}}
{{Listen|filename=America the Beautiful (US Navy Band).ogg|title="America the Beautiful"|description=Performed by the [[United States Navy Band]] and Sea Chanters, arr. Carmen Dragon|format=[[Ogg]]
|filename2=America (US Army Brass).ogg
|title2="America The Beautiful"
|description2=[[United States Army Band]] Brass version
|filetype2=[[Ogg]]
}}
 [[File:5 America, the Beautiful.png|right|thumb|300px|Sheet music version]]
"'''America the Beautiful'''" is an [[United States|American]] [[patriotic song]]. The lyrics were written by [[Katharine Lee Bates]] and the music composed by church organist and choirmaster [[Samuel A. Ward]].

Bates originally wrote the words as a poem, ''Pikes Peak'', first published in the [[Fourth of July]] edition of the church periodical ''The Congregationalist'' in 1895. At that time, the poem was titled ''America'' for publication.

Ward had originally written the music, ''Materna'', for the hymn ''[[O Mother dear, Jerusalem]]'' in 1882. Ward's music combined with the Bates poem was first published in 1910 and titled ''America the Beautiful''.

The song is one of the most beloved and popular of the many American patriotic songs.<ref>{{cite web|url=http://lcweb2.loc.gov/diglib/ihas/loc.natlib.ihas.100010615/full.html |title=Materna (O Mother Dear, Jerusalem) / Samuel Augustus Ward &#91;hymnal&#93;:Print Material Full Description: Performing Arts Encyclopedia, Library of Congress |publisher=Lcweb2.loc.gov |date=2007-10-30 |accessdate=2011-08-20}}</ref> From time to time it has been proposed as a replacement for ''[[The Star-Spangled Banner]]'' as the national anthem, including television [[sign-off]]s.{{citation needed|date=March 2012}}

==History==
In 1893, at the age of thirty-three, Bates, an English professor at [[Wellesley College]], had taken a train trip to [[Colorado Springs, Colorado|Colorado Springs]], Colorado, to teach a short summer school session at [[Colorado College]]. Several of the sights on her trip inspired her, and they found their way into her poem, including the [[World's Columbian Exposition]] in [[Chicago]], the "White City" with its promise of the future contained within its alabaster buildings; the wheat fields of America's heartland [[Kansas]], through which her train was riding on July 16; and the majestic view of the [[Great Plains]] from high atop Zebulon's [[Pikes Peak]].

On the pinnacle of that mountain, the words of the poem started to come to her, and she wrote them down upon returning to her hotel room at the original Antlers Hotel. The poem was initially published two years later in ''The Congregationalist,'' to commemorate the Fourth of July. It quickly caught the public's fancy. Amended versions were published in 1904 and 1913.

Several existing pieces of music were adapted to the poem. A [[hymn tune]] composed by Samuel A. Ward was generally considered the best music as early as 1910 and is still the popular tune today. Just as Bates had been inspired to write her poem, Ward too was inspired to compose his tune. The tune came to him while he was on a ferryboat trip from [[Coney Island]] back to his home in [[New York City]], after a leisurely summer day in 1882, and he immediately wrote it down. He was so anxious to capture the tune in his head, he asked fellow passenger friend Harry Martin for his shirt cuff to write the tune on. He composed the tune for the old hymn "O Mother Dear, Jerusalem", retitling the work "Materna". Ward's music combined with Bates' poem were first published together in 1910 and titled, ''America the Beautiful''.<ref name="Collins">{{cite book
 |title=Songs Sung Red, White, and Blue: The Stories Behind America's Best-Loved Patriotic Songs
 |first=Ace |last=Collins
 |page=19
 |publisher=Harper
 |year=2003
 |isbn= 0-06-051304-7
 |url=http://books.google.com/books?id=kKsA1Qi8BBgC&pg=PA19#v=onepage&f=false
}}</ref>

Ward died in 1903, not knowing the national stature his music would attain, as the music was only first applied to the song in 1904. Bates was more fortunate, as the song's popularity was well established by her death in 1929.

At various times in the more than 100 years that have elapsed since the song was written, particularly during the [[John F. Kennedy]] administration, there have been efforts to give "America the Beautiful" legal status either as a national hymn, or as a national anthem equal to, or in place of, "[[The Star-Spangled Banner]]", but so far this has not succeeded. Proponents prefer "America the Beautiful" for various reasons, saying it is easier to sing, more melodic, and more adaptable to new orchestrations while still remaining as easily recognizable as "The Star-Spangled Banner." Some prefer "America the Beautiful" over "The Star-Spangled Banner" due to the latter's war-oriented imagery. Others prefer "The Star-Spangled Banner" for the same reason. While that national dichotomy has stymied any effort at changing the tradition of the national anthem, "America the Beautiful" continues to be held in high esteem by a large number of Americans.

When [[Richard Nixon]] visited [[China]] in 1972, this song was played as the welcome music.

The song is often included in songbooks in a wide variety of [[religion|religious]] congregations in the United States.

== Lyrics ==
:O beautiful for spacious skies,
:For amber waves of grain,
:For purple mountain majesties
:Above the fruited plain!

:America! America!
:God shed His grace on thee,
:And crown thy good with brotherhood
:From sea to shining sea!

:O beautiful for pilgrim feet
:Whose stern impassion'd stress
:A thoroughfare for freedom beat
:Across the wilderness

:America! America!
:God mend thine ev'ry flaw,
:Confirm thy soul in self-control,
:Thy liberty in law.

:O beautiful for heroes prov'd
:In liberating strife,
:Who more than self their country lov'd,
:And mercy more than life.

:America! America!
:May God thy gold refine
:Till all success be nobleness,
:And ev'ry gain divine.

:O beautiful for patriot dream
:That sees beyond the years
:Thine alabaster cities gleam
:Undimmed by human tears.

:America! America!
:God shed His grace on thee,
:And crown thy good with brotherhood
:From sea to shining sea.

== Original poem (1893) ==
:'''America. A Poem for July 4.'''

:O beautiful for halcyon skies,
: For amber waves of grain,
:For purple mountain majesties
: Above the enameled plain!
:America! America!
: God shed His grace on thee,
:Till souls wax fair as earth and air
: And music-hearted sea!

:O beautiful for pilgrim feet
: Whose stern, impassioned stress
:A thoroughfare for freedom beat
: Across the wilderness!
:America! America!
:God shed His grace on thee
: Till paths be wrought through wilds of thought
:By pilgrim foot and knee!

:O beautiful for glory-tale
: Of liberating strife,
:When once or twice, for man's avail,
: Men lavished precious life!
:America! America!
: God shed His grace on thee
:Till selfish gain no longer stain,
: The banner of the free!

:O beautiful for patriot dream
: That sees beyond the years
:Thine alabaster cities gleam
: Undimmed by human tears!
:America! America!
: God shed His grace on thee
:Till nobler men keep once again
: Thy whiter jubilee!

==Popular versions==
Possibly the most notable current version of "America the Beautiful" is the setting for band and symphonic orchestra by the late arranger/conductor/composer [[Carmen Dragon]].  Performed thousands of times in recent years, Mr. Dragon's arrangement has been played for state occasions such as the memorial services for Presidents Ford and Reagan and at tribute concerts for events such as the Oklahoma City bombing and 9/11.

Three different renditions of the song have entered the [[Hot Country Songs]] charts. The first was by [[Charlie Rich]], which went to number 22 in 1976.<ref name="whitburn">{{cite book|last=Whitburn|first=Joel|title=Hot Country Songs 1944 to 2008|publisher=Record Research, Inc|year=2008|page=350|isbn=0-89820-177-2}}</ref> A second, by [[Mickey Newbury]], peaked at number 82 in 1980.<ref>Whitburn, p. 297</ref> An [[all-star]] version of "America the Beautiful" performed by [[country music|country]] singers [[Trace Adkins]], [[Billy Dean]], [[Vince Gill]], [[Carolyn Dawn Johnson]], [[Toby Keith]], [[Brenda Lee]], [[Lonestar]], [[Martina McBride]], [[Jamie O'Neal]], [[Kenny Rogers]] and [[Keith Urban]] reached number 58 in July 2001. The song re-entered the chart following the September 11 terrorist attacks.<ref>Whitburn, p. 24</ref>

Popularity of the song increased greatly following the [[September 11 attacks|September 11, 2001 attacks]]; at some sporting events it was sung in addition to the traditional singing of the national anthem.  During the first taping of the ''[[Late Show with David Letterman]]'' following the attacks, CBS newsman [[Dan Rather]] cried briefly as he quoted the fourth verse.<ref>{{cite news |date=2001-09-18 |work=Salon.com |title=Dan Rather's tears; Journalists don't cry on camera. That was before last week. |url=http://archive.salon.com/ent/tv/feature/2001/09/18/rather/index.html |first=Stephanie |last=Zacharek}}</ref>

[[Ray Charles]] is credited with the song's most well known rendition in current times (although [[Elvis Presley]] had success with it in the 1970s). Charles' recording is very commonly played at major sporting and entertainment events, such as the [[Super Bowl]], and [[WrestleMania]]; Charles gave a live performance of the song prior to [[Super Bowl XXXV]], the last Super Bowl played before the September 11 terrorist attacks. He places the third verse first, after which he sings the usual first verse. In the third verse (see above), the author scolds the materialistic and self-serving [[robber baron (industrialist)|robber barons]] of her day, and urges America to live up to its noble ideals and to honor, with both word and deed, the memory of those who died for their country. He also performed the song on Red Sox opening day at Fenway Park in 2003, though the game was eventually [[Rainout (sports)|rained out]].  A version with [[Alicia Keys]] was included in Charles' 2005 duets album ''[[Genius & Friends]]''.

Symbolically, [[Marian Anderson]] (a noted opera singer of her day) sang a rendition of "America the Beautiful" on the steps of the [[Lincoln Memorial]] in 1939 after being refused use of Constitution Hall by the [[Daughters of the American Revolution]] because of her skin color.

In 2009, on her album ''Running for the Drum'', [[Buffy Sainte-Marie]] released a new version of "America the Beautiful".  Her version contained some new lyrics, as well as a small modification of the melody.  This version includes only two verses, the second of which is original.

:O beautiful for vision clear,
:That sees beyond the years,
:The nightime sky, our hopes that fly,
:Undimmed by human tears.

:America! America!
:God shed His grace on thee,
:'Til selfish gain no longer stain,
:The banner of the free.

:And crown thy good with brotherhood,
:From sea to shining sea.

Wellesley students honor Bates with a version at graduation, substituting
:And crown thy good with SISTERhood,
:From sea to shining sea.

In 2008 [[Christina Aguilera]] covered this song for a [[Rock the Vote]] commercial

==Idioms==
'''"From sea to shining sea"''' is an [[United States|American]] [[idiom]] meaning from the [[Atlantic Ocean]] to the [[Pacific Ocean]] (or vice versa).  Many songs have used this term, including the American patriotic songs "America, The Beautiful" and "[[God Bless the USA]]".  In addition to these, it is also featured in [[Schoolhouse Rock]]'s "Elbow Room".  A term similar to this is the [[Canada|Canadian]] motto ''[[A Mari Usque Ad Mare]]'' ("From sea to sea.")

==Books==
*[[Lynn Sherr]]'s 2001 book ''America the Beautiful'' discusses the origins of the song and the backgrounds of its authors in depth. The book points out that the poem has the same meter as that of "[[Auld Lang Syne]]"; the songs can be sung interchangeably. Additionally, Sherr points out that this was the original third verse written by Bates:<ref name="Sherr">{{cite book|title=America the Beautiful: The Stirring True Story Behind Our Nation's Favorite Song|first=Lynn|last=Sherr|publisher=PublicAffairs
 |year=2001
 |isbn= 978-1-58648-085-1
 |url=
}}</ref>

:America! America!
:God shed His grace on thee,
:'Til selfish gain no longer stain,
:The banner of the free.

*Barbara Younger has written a children's book about the writing of the song: ''Purple Mountain Majesties: The Story of Katharine Lee Bates and "America the Beautiful"''. The book has illustrations by artist Stacey Schuett.

==References==
{{Reflist}}
Sherr, Lynn (2001). ''America the Beautiful: The Stirring True Story Behind Our Nation's Favorite Song''. PublicAffairs. ISBN 978-1-58648-085-1.

==Sources/external links==
{{Wikisource}}
*[http://memory.loc.gov/cocoon/ihas/loc.natlib.ihas.200000001/default.html MP3 and RealAudio recordings available at the United States Library of Congress]
*{{cantorion|pieces/3049/America_the_Beautiful_(Materna)|America the Beautiful}}
*[http://hymntime.com/tch/htm/o/b/f/obfsskis.htm Words, sheet music & MIDI file] at the Cyber Hymnal
*[http://www.springsgov.com/Page.asp?NavID=5138 America the Beautiful Park] in Colorado Springs named for Katharine Lee Bates' words.
*[http://web.me.com/highlandsarchives/The_Highlands_Archives/America.html  Archival collection] of America the Beautiful lantern slides from the 1930s.
*[http://www.traditional-songs.com/download_score.php?name=America%20the%20Beautiful&country=Usa Another free sheet music]
*[http://loyalopposition.blogs.nytimes.com/2012/01/03/mitt-romney-and-america-the-beautiful-when-reach-exceeds-grasp/?ref=opinion David Firestone - When Romney’s Reach Exceeds His Grasp - Mitt Romney quotes the song ]

{{Elvis Presley singles}}

{{DEFAULTSORT:America The Beautiful}}
[[Category:American patriotic songs]]
[[Category:1895 songs]]
[[Category:Pikes Peak]]
[[Category:Elvis Presley songs]]
[[Category:Charlie Rich songs]]
[[Category:Ray Charles songs]]
[[Category:Barbra Streisand songs]]
[[Category:Mormon Tabernacle Choir songs]]

[[de:America the Beautiful]]
[[es:America The Beautiful]]
[[fr:America the Beautiful (hymne)]]
[[nl:America the Beautiful]]
[[pl:America the Beautiful]]
[[pt:America the Beautiful]]
[[ru:America the Beautiful]]
[[vi:America the Beautiful]]
[[zh:美丽的阿美利加]]