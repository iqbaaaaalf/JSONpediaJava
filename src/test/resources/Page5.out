Begin Document
Begin Template: Redirect
k: National Hymn
k: the hymn tune of that title
k: God of Our Fathers
End Template: Redirect
Text: '
'
Begin Template: Two other uses
k: an American patriotic song
k: the US coin series
k: America the Beautiful Quarters
k: the documentary film
k: America the Beautiful (film)
End Template: Two other uses
Text: '
'
Begin Template: Infobox song <!-- See Wikipedia:WikiProject_Songs -->

k: Name           
Text: 'America the Beautiful
'
k: Cover          
Text: 'Americathebeautiful.jpg
'
k: Caption        
Text: 'Commemoration plaque atop '
Reference: Pikes Peak ''
Text: '
'
k: Type           
Reference: Patriotic song 'Patriotic'
Text: '
'
k: Artist         
k: alt Artist     
k: Album          
k: Published      
Text: '1895 (poem)
'
k: Released       
k: track_no       
k: Recorded       
k: Genre          
k: Length         
k: Writer         
Reference: Katharine Lee Bates ''
Text: '
'
k: Composer       
Reference: Samuel A. Ward ''
Text: '
'
k: Label          
k: Producer       
k: Audio sample?  
Text: 'yes
'
k: Chart position 
k: Tracks         
k: prev           
k: prev_no        
k: next           
k: next_no        
k: Misc           
Begin Template: Audiosample

k: Upper caption 
Text: 'America the Beautiful MIDI recording.
'
k: Audio file    
Text: 'America the Beautiful.mid
'
k: Background    
Text: 'Song
'
End Template: Audiosample

Text: '
'
End Template: Infobox song <!-- See Wikipedia:WikiProject_Songs -->

Text: '
'
Begin Template: Listen
k: filename
Text: 'America the Beautiful (US Navy Band).ogg'
k: title
Text: '"America the Beautiful"'
k: description
Text: 'Performed by the '
Reference: United States Navy Band ''
Text: ' and Sea Chanters, arr. Carmen Dragon'
k: format
Reference: Ogg ''
Text: '
'
k: filename2
Text: 'America (US Army Brass).ogg
'
k: title2
Text: '"America The Beautiful"
'
k: description2
Reference: United States Army Band ''
Text: ' Brass version
'
k: filetype2
Reference: Ogg ''
Text: '
'
End Template: Listen
Text: '
 '
Reference: File:5 America, the Beautiful.png 'right|thumb|300px|Sheet music version'
Text: '
"'''America the Beautiful'''" is an '
Reference: United States 'American'
Text: ' '
Reference: patriotic song ''
Text: '. The lyrics were written by '
Reference: Katharine Lee Bates ''
Text: ' and the music composed by church organist and choirmaster '
Reference: Samuel A. Ward ''
Text: '.

Bates originally wrote the words as a poem, ''Pikes Peak'', first published in the '
Reference: Fourth of July ''
Text: ' edition of the church periodical ''The Congregationalist'' in 1895. At that time, the poem was titled ''America'' for publication.

Ward had originally written the music, ''Materna'', for the hymn '''
Reference: O Mother dear, Jerusalem ''
Text: ''' in 1882. Ward's music combined with the Bates poem was first published in 1910 and titled ''America the Beautiful''.

The song is one of the most beloved and popular of the many American patriotic songs.'
Open Tag: ref attributes: []
Begin Template: cite web
k: url
Text: 'http://lcweb2.loc.gov/diglib/ihas/loc.natlib.ihas.100010615/full.html '
k: title
Text: 'Materna (O Mother Dear, Jerusalem) / Samuel Augustus Ward '
Text: 'hymnal&#93;:Print'
Text: ' '
Text: 'Material Full Description: Performing Arts Encyclopedia, Library of Congress '
k: publisher
Text: 'Lcweb2.loc.gov '
k: date
Text: '2007-10-30 '
k: accessdate
Text: '2011-08-20'
End Template: cite web
Close Tag: ref
Text: ' From time to time it has been proposed as a replacement for '''
Reference: The Star-Spangled Banner ''
Text: ''' as the national anthem, including television '
Reference: sign-off ''
Text: 's.'
Begin Template: citation needed
k: date
Text: 'March 2012'
End Template: citation needed
Text: '

'
Section [0] History
Text: '
In 1893, at the age of thirty-three, Bates, an English professor at '
Reference: Wellesley College ''
Text: ', had taken a train trip to '
Reference: Colorado Springs, Colorado 'Colorado Springs'
Text: ', Colorado, to teach a short summer school session at '
Reference: Colorado College ''
Text: '. Several of the sights on her trip inspired her, and they found their way into her poem, including the '
Reference: World's Columbian Exposition ''
Text: ' in '
Reference: Chicago ''
Text: ', the "White City" with its promise of the future contained within its alabaster buildings; the wheat fields of America's heartland '
Reference: Kansas ''
Text: ', through which her train was riding on July 16; and the majestic view of the '
Reference: Great Plains ''
Text: ' from high atop Zebulon's '
Reference: Pikes Peak ''
Text: '.

On the pinnacle of that mountain, the words of the poem started to come to her, and she wrote them down upon returning to her hotel room at the original Antlers Hotel. The poem was initially published two years later in ''The Congregationalist,'' to commemorate the Fourth of July. It quickly caught the public's fancy. Amended versions were published in 1904 and 1913.

Several existing pieces of music were adapted to the poem. A '
Reference: hymn tune ''
Text: ' composed by Samuel A. Ward was generally considered the best music as early as 1910 and is still the popular tune today. Just as Bates had been inspired to write her poem, Ward too was inspired to compose his tune. The tune came to him while he was on a ferryboat trip from '
Reference: Coney Island ''
Text: ' back to his home in '
Reference: New York City ''
Text: ', after a leisurely summer day in 1882, and he immediately wrote it down. He was so anxious to capture the tune in his head, he asked fellow passenger friend Harry Martin for his shirt cuff to write the tune on. He composed the tune for the old hymn "O Mother Dear, Jerusalem", retitling the work "Materna". Ward's music combined with Bates' poem were first published together in 1910 and titled, ''America the Beautiful''.'
Open Tag: ref attributes: [name : 'Collins']
Begin Template: cite book
 
k: title
Text: 'Songs Sung Red, White, and Blue: The Stories Behind America's Best-Loved Patriotic Songs
 '
k: first
Text: 'Ace '
k: last
Text: 'Collins
 '
k: page
Text: '19
 '
k: publisher
Text: 'Harper
 '
k: year
Text: '2003
 '
k: isbn
Text: '0-06-051304-7
 '
k: url
Text: 'http://books.google.com/books?id=kKsA1Qi8BBgC&pg=PA19#v=onepage&f=false
'
End Template: cite book
 
Close Tag: ref
Text: '

Ward died in 1903, not knowing the national stature his music would attain, as the music was only first applied to the song in 1904. Bates was more fortunate, as the song's popularity was well established by her death in 1929.

At various times in the more than 100 years that have elapsed since the song was written, particularly during the '
Reference: John F. Kennedy ''
Text: ' administration, there have been efforts to give "America the Beautiful" legal status either as a national hymn, or as a national anthem equal to, or in place of, "'
Reference: The Star-Spangled Banner ''
Text: '", but so far this has not succeeded. Proponents prefer "America the Beautiful" for various reasons, saying it is easier to sing, more melodic, and more adaptable to new orchestrations while still remaining as easily recognizable as "The Star-Spangled Banner." Some prefer "America the Beautiful" over "The Star-Spangled Banner" due to the latter's war-oriented imagery. Others prefer "The Star-Spangled Banner" for the same reason. While that national dichotomy has stymied any effort at changing the tradition of the national anthem, "America the Beautiful" continues to be held in high esteem by a large number of Americans.

When '
Reference: Richard Nixon ''
Text: ' visited '
Reference: China ''
Text: ' in 1972, this song was played as the welcome music.

The song is often included in songbooks in a wide variety of '
Reference: religion 'religious'
Text: ' congregations in the United States.

'
Section [0]  Lyrics 
Text: '
:O beautiful for spacious skies,
:For amber waves of grain,
:For purple mountain majesties
:Above the fruited plain!

:America! America!
:God shed His grace on thee,
:And crown thy good with brotherhood
:From sea to shining sea!

:O beautiful for pilgrim feet
:Whose stern impassion'd stress
:A thoroughfare for freedom beat
:Across the wilderness

:America! America!
:God mend thine ev'ry flaw,
:Confirm thy soul in self-control,
:Thy liberty in law.

:O beautiful for heroes prov'd
:In liberating strife,
:Who more than self their country lov'd,
:And mercy more than life.

:America! America!
:May God thy gold refine
:Till all success be nobleness,
:And ev'ry gain divine.

:O beautiful for patriot dream
:That sees beyond the years
:Thine alabaster cities gleam
:Undimmed by human tears.

:America! America!
:God shed His grace on thee,
:And crown thy good with brotherhood
:From sea to shining sea.

'
Section [0]  Original poem (1893) 
Text: '
:'''America. A Poem for July 4.'''

:O beautiful for halcyon skies,
: For amber waves of grain,
:For purple mountain majesties
: Above the enameled plain!
:America! America!
: God shed His grace on thee,
:Till souls wax fair as earth and air
: And music-hearted sea!

:O beautiful for pilgrim feet
: Whose stern, impassioned stress
:A thoroughfare for freedom beat
: Across the wilderness!
:America! America!
:God shed His grace on thee
: Till paths be wrought through wilds of thought
:By pilgrim foot and knee!

:O beautiful for glory-tale
: Of liberating strife,
:When once or twice, for man's avail,
: Men lavished precious life!
:America! America!
: God shed His grace on thee
:Till selfish gain no longer stain,
: The banner of the free!

:O beautiful for patriot dream
: That sees beyond the years
:Thine alabaster cities gleam
: Undimmed by human tears!
:America! America!
: God shed His grace on thee
:Till nobler men keep once again
: Thy whiter jubilee!

'
Section [0] Popular versions
Text: '
Possibly the most notable current version of "America the Beautiful" is the setting for band and symphonic orchestra by the late arranger/conductor/composer '
Reference: Carmen Dragon ''
Text: '.  Performed thousands of times in recent years, Mr. Dragon's arrangement has been played for state occasions such as the memorial services for Presidents Ford and Reagan and at tribute concerts for events such as the Oklahoma City bombing and 9/11.

Three different renditions of the song have entered the '
Reference: Hot Country Songs ''
Text: ' charts. The first was by '
Reference: Charlie Rich ''
Text: ', which went to number 22 in 1976.'
Open Tag: ref attributes: [name : 'whitburn']
Begin Template: cite book
k: last
Text: 'Whitburn'
k: first
Text: 'Joel'
k: title
Text: 'Hot Country Songs 1944 to 2008'
k: publisher
Text: 'Record Research, Inc'
k: year
Text: '2008'
k: page
Text: '350'
k: isbn
Text: '0-89820-177-2'
End Template: cite book
Close Tag: ref
Text: ' A second, by '
Reference: Mickey Newbury ''
Text: ', peaked at number 82 in 1980.'
Open Tag: ref attributes: []
Text: 'Whitburn, p. 297'
Close Tag: ref
Text: ' An '
Reference: all-star ''
Text: ' version of "America the Beautiful" performed by '
Reference: country music 'country'
Text: ' singers '
Reference: Trace Adkins ''
Text: ', '
Reference: Billy Dean ''
Text: ', '
Reference: Vince Gill ''
Text: ', '
Reference: Carolyn Dawn Johnson ''
Text: ', '
Reference: Toby Keith ''
Text: ', '
Reference: Brenda Lee ''
Text: ', '
Reference: Lonestar ''
Text: ', '
Reference: Martina McBride ''
Text: ', '
Reference: Jamie O'Neal ''
Text: ', '
Reference: Kenny Rogers ''
Text: ' and '
Reference: Keith Urban ''
Text: ' reached number 58 in July 2001. The song re-entered the chart following the September 11 terrorist attacks.'
Open Tag: ref attributes: []
Text: 'Whitburn, p. 24'
Close Tag: ref
Text: '

Popularity of the song increased greatly following the '
Reference: September 11 attacks 'September 11, 2001 attacks'
Text: '; at some sporting events it was sung in addition to the traditional singing of the national anthem.  During the first taping of the '''
Reference: Late Show with David Letterman ''
Text: ''' following the attacks, CBS newsman '
Reference: Dan Rather ''
Text: ' cried briefly as he quoted the fourth verse.'
Open Tag: ref attributes: []
Begin Template: cite news 
k: date
Text: '2001-09-18 '
k: work
Text: 'Salon.com '
k: title
Text: 'Dan Rather's tears; Journalists don't cry on camera. That was before last week. '
k: url
Text: 'http://archive.salon.com/ent/tv/feature/2001/09/18/rather/index.html '
k: first
Text: 'Stephanie '
k: last
Text: 'Zacharek'
End Template: cite news 
Close Tag: ref
Text: '

'
Reference: Ray Charles ''
Text: ' is credited with the song's most well known rendition in current times (although '
Reference: Elvis Presley ''
Text: ' had success with it in the 1970s). Charles' recording is very commonly played at major sporting and entertainment events, such as the '
Reference: Super Bowl ''
Text: ', and '
Reference: WrestleMania ''
Text: '; Charles gave a live performance of the song prior to '
Reference: Super Bowl XXXV ''
Text: ', the last Super Bowl played before the September 11 terrorist attacks. He places the third verse first, after which he sings the usual first verse. In the third verse (see above), the author scolds the materialistic and self-serving '
Reference: robber baron (industrialist) 'robber barons'
Text: ' of her day, and urges America to live up to its noble ideals and to honor, with both word and deed, the memory of those who died for their country. He also performed the song on Red Sox opening day at Fenway Park in 2003, though the game was eventually '
Reference: Rainout (sports) 'rained out'
Text: '.  A version with '
Reference: Alicia Keys ''
Text: ' was included in Charles' 2005 duets album '''
Reference: Genius & Friends ''
Text: '''.

Symbolically, '
Reference: Marian Anderson ''
Text: ' (a noted opera singer of her day) sang a rendition of "America the Beautiful" on the steps of the '
Reference: Lincoln Memorial ''
Text: ' in 1939 after being refused use of Constitution Hall by the '
Reference: Daughters of the American Revolution ''
Text: ' because of her skin color.

In 2009, on her album ''Running for the Drum'', '
Reference: Buffy Sainte-Marie ''
Text: ' released a new version of "America the Beautiful".  Her version contained some new lyrics, as well as a small modification of the melody.  This version includes only two verses, the second of which is original.

:O beautiful for vision clear,
:That sees beyond the years,
:The nightime sky, our hopes that fly,
:Undimmed by human tears.

:America! America!
:God shed His grace on thee,
:'Til selfish gain no longer stain,
:The banner of the free.

:And crown thy good with brotherhood,
:From sea to shining sea.

Wellesley students honor Bates with a version at graduation, substituting
:And crown thy good with SISTERhood,
:From sea to shining sea.

In 2008 '
Reference: Christina Aguilera ''
Text: ' covered this song for a '
Reference: Rock the Vote ''
Text: ' commercial

'
Section [0] Idioms
Text: '
'''"From sea to shining sea"''' is an '
Reference: United States 'American'
Text: ' '
Reference: idiom ''
Text: ' meaning from the '
Reference: Atlantic Ocean ''
Text: ' to the '
Reference: Pacific Ocean ''
Text: ' (or vice versa).  Many songs have used this term, including the American patriotic songs "America, The Beautiful" and "'
Reference: God Bless the USA ''
Text: '".  In addition to these, it is also featured in '
Reference: Schoolhouse Rock ''
Text: ''s "Elbow Room".  A term similar to this is the '
Reference: Canada 'Canadian'
Text: ' motto '''
Reference: A Mari Usque Ad Mare ''
Text: ''' ("From sea to sea.")

'
Section [0] Books
Text: '
*'
Reference: Lynn Sherr ''
Text: ''s 2001 book ''America the Beautiful'' discusses the origins of the song and the backgrounds of its authors in depth. The book points out that the poem has the same meter as that of "'
Reference: Auld Lang Syne ''
Text: '"; the songs can be sung interchangeably. Additionally, Sherr points out that this was the original third verse written by Bates:'
Open Tag: ref attributes: [name : 'Sherr']
Begin Template: cite book
k: title
Text: 'America the Beautiful: The Stirring True Story Behind Our Nation's Favorite Song'
k: first
Text: 'Lynn'
k: last
Text: 'Sherr'
k: publisher
Text: 'PublicAffairs
 '
k: year
Text: '2001
 '
k: isbn
Text: '978-1-58648-085-1
 '
k: url
End Template: cite book
Close Tag: ref
Text: '

:America! America!
:God shed His grace on thee,
:'Til selfish gain no longer stain,
:The banner of the free.

*Barbara Younger has written a children's book about the writing of the song: ''Purple Mountain Majesties: The Story of Katharine Lee Bates and "America the Beautiful"''. The book has illustrations by artist Stacey Schuett.

'
Section [0] References
Text: '
'
Begin Template: Reflist
End Template: Reflist
Text: '
Sherr, Lynn (2001). ''America the Beautiful: The Stirring True Story Behind Our Nation's Favorite Song''. PublicAffairs. ISBN 978-1-58648-085-1.

'
Section [0] Sources/external links
Text: '
'
Begin Template: Wikisource
End Template: Wikisource
Text: '
*'
Link: http://memory.loc.gov/cocoon/ihas/loc.natlib.ihas.200000001/default.html 'MP3 and RealAudio recordings available at the United States Library of Congress'
Text: '
*'
Begin Template: cantorion
k: pieces/3049/America_the_Beautiful_(Materna)
k: America the Beautiful
End Template: cantorion
Text: '
*'
Link: http://hymntime.com/tch/htm/o/b/f/obfsskis.htm 'Words, sheet music & MIDI file'
Text: ' at the Cyber Hymnal
*'
Link: http://www.springsgov.com/Page.asp?NavID=5138 'America the Beautiful Park'
Text: ' in Colorado Springs named for Katharine Lee Bates' words.
*'
Link: http://web.me.com/highlandsarchives/The_Highlands_Archives/America.html ' Archival collection'
Text: ' of America the Beautiful lantern slides from the 1930s.
*'
Link: http://www.traditional-songs.com/download_score.php?name=America%20the%20Beautiful&country=Usa 'Another free sheet music'
Text: '
*'
Link: http://loyalopposition.blogs.nytimes.com/2012/01/03/mitt-romney-and-america-the-beautiful-when-reach-exceeds-grasp/?ref=opinion 'David Firestone - When Romney’s Reach Exceeds His Grasp - Mitt Romney quotes the song '
Text: '

'
Begin Template: Elvis Presley singles
End Template: Elvis Presley singles
Text: '

'
Begin Template: DEFAULTSORT:America The Beautiful
End Template: DEFAULTSORT:America The Beautiful
Text: '
'
Reference: Category:American patriotic songs ''
Text: '
'
Reference: Category:1895 songs ''
Text: '
'
Reference: Category:Pikes Peak ''
Text: '
'
Reference: Category:Elvis Presley songs ''
Text: '
'
Reference: Category:Charlie Rich songs ''
Text: '
'
Reference: Category:Ray Charles songs ''
Text: '
'
Reference: Category:Barbra Streisand songs ''
Text: '
'
Reference: Category:Mormon Tabernacle Choir songs ''
Text: '

'
Reference: de:America the Beautiful ''
Text: '
'
Reference: es:America The Beautiful ''
Text: '
'
Reference: fr:America the Beautiful (hymne) ''
Text: '
'
Reference: nl:America the Beautiful ''
Text: '
'
Reference: pl:America the Beautiful ''
Text: '
'
Reference: pt:America the Beautiful ''
Text: '
'
Reference: ru:America the Beautiful ''
Text: '
'
Reference: vi:America the Beautiful ''
Text: '
'
Reference: zh:美丽的阿美利加 ''
End Document
